### ISSUE TITLE
[question]: good pause button

### ISSUE THREAD

**Issue by shinriyo** [RafaelBarbosatec/bonfire#465]
### How can we help?

I modified the sample game like below.

`KnightInterface` class.

```
    await add(InterfaceComponent(
      spriteUnselected: Sprite.load('blue_button1.png'),
      spriteSelected: Sprite.load('blue_button2.png'),
      size: Vector2(40, 40),
      id: 5,
      position: Vector2(300, 20),
      selectable: false,
      onTapComponent: (selected) {
        if (hasGameRef) {
          if (isPause) { // how to get?
            gameRef.resumeEngine();
          } else {
            gameRef.pauseEngine();
          }
        }
      },
    ));
```

1.
I want to know if the game is paused or not.

2.

I want to know how to avoid stopping the button animation when game paused.

**Comment by RafaelBarbosatec** [comment]
Hi @shinriyo !
Good observation. There isn’t a way to check it. I will adds it.

**Comment by shinriyo** [comment]
Thank you!

**Comment by RafaelBarbosatec** [comment]
Available in `3.0.6`
